src/components/PadSettings.spec.js

```diff
@@ -155,15 +156,64 @@ describe('PadSettings', () => {
         />
       </MemoryRouter>
     );
-    const padSelect = screen.getByRole('combobox', { name: 'select a pad' })
-    fireEvent.change(padSelect)
-    expect(padChange).toHaveBeenCalled()
-    const colorSelect = screen.getByRole('combobox', { name: 'select a color' })
-    fireEvent.change(colorSelect)
-    expect(colorChange).toHaveBeenCalled()
-    const sampleSelect = screen.getByRole('combobox', { name: 'select a sample' })
-    fireEvent.change(sampleSelect)
-    expect(sampleChange).toHaveBeenCalled()
+    const padSelect = screen.getByRole('combobox', { name: 'select a pad' });
+    fireEvent.change(padSelect);
+    expect(padChange).toHaveBeenCalled();
+    const colorSelect = screen.getByRole('combobox', {
+      name: 'select a color',
+    });
+    fireEvent.change(colorSelect);
+    expect(colorChange).toHaveBeenCalled();
+    const sampleSelect = screen.getByRole('combobox', {
+      name: 'select a sample',
+    });
+    fireEvent.change(sampleSelect);
+    expect(sampleChange).toHaveBeenCalled();
+  });
```

Comment:
As a fellow developer, it is hard to distinguish what you are searching in your rendered screen and what you are expecting as a result; maybe it's better to distinguish the variable declarations and the `expect` expressions?

Reply:
Good Idea, I will change this 👍🏻